controller/structures.xml

```diff
@@ -41,6 +34,9 @@
                             context="resource" group="tree" binding="removeNode">
                         <icon id="icon-bin"/>
                     </action>
+                    <action id="testcenter-delete-all" name="Delete" url="/taoTestCenter/TestCenterManager/deleteAll" multiple="true" context="resource" group="tree" binding="removeNodes">
```

Comment:
The display name of 'delete all' is the same as for the 'delete one'.

Reply:
They aren't supposed to be displayed at the same time, multiple and single contexts are exclusives. The bug is fixed in tao-core